### PR TITLE
Added clipboard CB_HUGE_FILE_SUPPORT_ENABLED flag

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

**Comment by mfleisz** [comment]
@freerdp-bot test

File: include/freerdp/channels/cliprdr.h [hunk 1]
```diff
@@ -70,6 +70,7 @@
 #define CB_STREAM_FILECLIP_ENABLED 0x00000004
 #define CB_FILECLIP_NO_FILE_PATHS 0x00000008
 #define CB_CAN_LOCK_CLIPDATA 0x00000010
+#define CB_HUGE_FILE_SUPPORT_ENABLED 0x00000010
```
Comment (mfleisz) [inline include/freerdp/channels/cliprdr.h:73]: I think this should be `0x00000020`

Comment (akallabeth) [reply include/freerdp/channels/cliprdr.h]: yes, fixes.

**Review by mfleisz — APPROVED** [review]
LGTM

**Pull request merged**